zhaquirks/tuya/ts0601_sensor.py

```diff
@@ -1,487 +1,67 @@
-                self.endpoint.device.RH_MULTIPLIER
-                if hasattr(self.endpoint.device, "RH_MULTIPLIER")
-                else 100
-            )
-        return super().update_attribute(attr_name, value)
-
-
-class TemperatureHumidityManufCluster(TuyaMCUCluster):
-    """Tuya Manufacturer Cluster with Temperature and Humidity data points."""
-
-    dp_to_attribute: dict[int, DPToAttributeMapping] = {
-        1: DPToAttributeMapping(
-            TuyaTemperatureMeasurement.ep_attribute,
-            "measured_value",
-            converter=lambda x: x * 10,  # decidegree to centidegree
-        ),
-        2: DPToAttributeMapping(
-            TuyaRelativeHumidity.ep_attribute,
-            "measured_value",
-            # converter=lambda x: x * 10,  --> move conversion to TuyaRelativeHumidity cluster
-        ),
-        4: DPToAttributeMapping(
-            TuyaPowerConfigurationCluster2AAA.ep_attribute,
-            "battery_percentage_remaining",
-            converter=lambda x: x * 2,  # double reported percentage
-        ),
-    }
-
-    data_point_handlers = {
-        1: "_dp_2_attr_update",
-        2: "_dp_2_attr_update",
-        4: "_dp_2_attr_update",
-    }
-
-
-class TemperatureHumidityBatteryStatesManufCluster(TuyaMCUCluster):
-    """Tuya Manufacturer Cluster with Temperature and Humidity data points. Battery states 25, 50 and 100%."""
-
-    dp_to_attribute: dict[int, DPToAttributeMapping] = {
-        1: TemperatureHumidityManufCluster.dp_to_attribute[1],
-        2: TemperatureHumidityManufCluster.dp_to_attribute[2],
-        3: DPToAttributeMapping(
-            TuyaPowerConfigurationCluster2AAA.ep_attribute,
-            "battery_percentage_remaining",
-            converter=lambda x: {0: 50, 1: 100, 2: 200}[x],
-        ),
-    }
-
-    data_point_handlers = {
-        1: "_dp_2_attr_update",
-        2: "_dp_2_attr_update",
-        3: "_dp_2_attr_update",
-    }
-
-
-class TuyaTempHumiditySensor(CustomDevice):
-    """Custom device representing tuya temp and humidity sensor with e-ink screen."""
-
-    # RelativeHumidity multiplier
-    RH_MULTIPLIER = 10
-
-    signature = {
-        # <SimpleDescriptor endpoint=1, profile=260, device_type=81
-        # device_version=1
-        # input_clusters=[4, 5, 61184, 0]
-        # output_clusters=[25, 10]>
-        MODELS_INFO: [
-            ("_TZE200_bjawzodf", "TS0601"),
-            ("_TZE200_zl1kmjqx", "TS0601"),
-        ],
-        ENDPOINTS: {
-            1: {
-                PROFILE_ID: zha.PROFILE_ID,
-                DEVICE_TYPE: zha.DeviceType.SMART_PLUG,
-                INPUT_CLUSTERS: [
-                    Basic.cluster_id,
-                    Groups.cluster_id,
-                    Scenes.cluster_id,
-                    TemperatureHumidityManufCluster.cluster_id,
-                ],
-                OUTPUT_CLUSTERS: [Ota.cluster_id, Time.cluster_id],
-            }
-        },
-    }
-
-    replacement = {
-        SKIP_CONFIGURATION: True,
```

Comment:
Do we want to include `.skip_configuration()`?
By default, ZHA binds some default clusters like `PowerConfiguration` and `Scenes`(?) and sets up reporting (if they have a `REPORT_CONFIG`.
Most of our replacement clusters are `LocalDataCluster`s anyway, so it doesn't matter for them, as they ignore the calls.
By including `.skip_configuration()`, we'd make sure that ZHA doesn't even try to bind any clusters. Do we want that? The v1 quirks seem to have it.

Reply:
I'll add it, I never saw anything in the logs, so I don't think it matters, but makes sense to be clear.